pincer/utils/shards.py

```diff
@@ -0,0 +1,14 @@
+# Copyright Pincer 2021-Present
+# Full MIT License can be found in `LICENSE` at the project root.
+
+def calculate_shard_id(guild_id, num_shards):
```

Comment:
*May need to import Snowflake*
```suggestion
def calculate_shard_id(guild_id: Union[Snowflake, int], num_shards: int) -> int:
```